packages/pretty-format/src/__tests__/react.test.js

```diff
@@ -90,6 +91,13 @@ test('supports a single element with mixed children', () => {
   );
 });
 
+test('supports a single fragment with one child', () => {
```

Comment:
This test fails because we're on React 15

Reply:
The alternatives are:

Reply:
We should update to latest React